frontend/cypress/tests/codeView/codeView_renderFilterHash.cy.js

```diff
@@ -309,31 +309,184 @@ describe('render filter hash', () => {
+      .should('contain', 'yml');
+
     cy.get('#summary div.fileTypes input[id="gradle"]')
       .uncheck()
       .should('not.be.checked');
 
     cy.url()
       .should('not.contain', 'gradle');
 
+    cy.url()
+      .should('contain', 'yml');
+
     cy.reload();
 
+    cy.get('#summary div.fileTypes input[id="gradle"]')
+      .should('not.be.checked');
+
+    cy.get('#summary div.fileTypes input[id="yml"]')
+      .should('be.checked');
+
     cy.url()
       .should('not.contain', 'gradle');
+
+    cy.url()
+      .should('contain', 'yml');
+  });
+
+  it('code panel: sort by: url params should persist after change and reload', () => {
+    // open the code panel
+    cy.get('.icon-button.fa-code')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('div.mui-select.sort-by > select:visible')
+      .invoke('val')
+      .should('eq', 'linesOfCode');
+
+    cy.url()
+      .should('not.contain', 'authorshipSortBy');
+
+    /* Select file name and test URL before and after reload */
+    cy.get('div.mui-select.sort-by > select:visible')
+      .select('fileName');
+
+    cy.url()
+      .should('contain', 'authorshipSortBy=fileName');
+
+    cy.reload();
+
+    cy.url()
+      .should('not.contain', '%23%2F');
+
+    cy.url()
+      .should('contain', 'authorshipSortBy=fileName');
+
+    /* Select file type and test URL before and after reload */
+    cy.get('div.mui-select.sort-by > select:visible')
+      .select('fileType');
+
+    cy.url()
+      .should('contain', 'authorshipSortBy=fileType');
+
+    cy.reload();
+
+    cy.url()
+      .should('not.contain', '%23%2F');
+
+    cy.url()
+      .should('contain', 'authorshipSortBy=fileType');
+  });
+
+  it('code panel: order: url params should persist after change and reload', () => {
+    // open the code panel
+    cy.get('.icon-button.fa-code')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('div.mui-select.sort-order > select:visible')
+      .invoke('val')
+      .should('eq', 'true'); // true is Descending
+
+    cy.url()
+      .should('not.contain', 'reverseAuthorshipOrder');
+
+    /* Select ascending and test URL before and after reload */
+    cy.get('div.mui-select.sort-order > select:visible')
+      .select('false');
+
+    cy.url()
+      .should('contain', 'reverseAuthorshipOrder=false');
+
+    cy.reload();
+
+    cy.url()
+      .should('not.contain', '%23%2F');
+
+    cy.url()
+      .should('contain', 'reverseAuthorshipOrder=false');
+
+    /* Select descending and test URL before and after reload */
+
+    cy.get('div.mui-select.sort-order > select:visible')
+      .select('true');
+
+    cy.url()
+      .should('contain', 'reverseAuthorshipOrder=true');
+
+    cy.reload();
+
+    cy.url()
+      .should('not.contain', '%23%2F');
+
+    cy.url()
+      .should('contain', 'reverseAuthorshipOrder=true');
+  });
+
+  it('code panel: filter by glob: url params should persist after change and reload', () => {
```

Comment:
This function only tests after one reload, but there is a bug in the filter by glob string after 2 reloads. 
Hence, just wanted to add a note that we must add a test with 2 page reloads when the bug in #1987 gets fixed.